spark/src/db/teams.py

```diff
@@ -0,0 +1,15 @@
+from src.db.RDBM import RethinkDBManager
+
+def update_or_insert_teams(players):    
```

Comment:
Can you please rename `players` to `teams` ? It is kind of confusing.
Also, can you juste merge `update_or_insert_players` and `update_or_insert_teams` into a single utility function `update_or_insert` that takes into account an additional argument: the table to update.
These 2 functions are really similar.

Reply:
They are similar now because we only have one field and two fields in the tables but in the future they are going to be different